dotnet/samples/KernelSyntaxExamples/Example09_FunctionTypes.cs

```diff
@@ -95,109 +86,60 @@ public async Task<string> Type03Async()
     }
 
     [KernelFunction]
-    public void Type04(ContextVariables variables)
-    {
-        Console.WriteLine("Running function type 4");
-    }
-
-    [KernelFunction]
-    public string Type05(ContextVariables variables)
-    {
-        Console.WriteLine("Running function type 5");
-        return "";
-    }
-
-    [KernelFunction]
-    public async Task<string> Type06Async(Kernel kernel)
+    public async Task<string> Type04Async(Kernel kernel)
     {
-        var summary = await kernel.InvokeAsync(kernel.Plugins["SummarizePlugin"]["Summarize"], new ContextVariables("blah blah blah"));
-        Console.WriteLine($"Running function type 6 [{summary?.GetValue<string>()}]");
+        var summary = await kernel.InvokeAsync(kernel.Plugins["SummarizePlugin"]["Summarize"], new KernelFunctionArguments() { { "input", "blah blah blah" } });
```

Comment:
If we're passing this to more than just KernelFunction.InvokeAsync, I wonder if it should be KernelArguments rather than KernelFunctionArguments?  Not for this PR, just something to think about as a possible refactor immediately after.

Reply:
Yep, good idea, in this case the class could be used in a broader context.

Reply:
This will be addressed in scope of this issue - https://github.com/microsoft/semantic-kernel/issues/3761 shortly after the PR is merged.